bindings/electron/src/lib.rs

```diff
@@ -10,6 +10,37 @@ lazy_static::lazy_static! {
         Mutex::new(tokio::runtime::Runtime::new().expect("Cannot start tokio runtime for libparsec"));
 }
 
+static BACKTRACE: Mutex<Option<Backtrace>> = Mutex::new(None);
+
+macro_rules! capture_backtrace {
+    ($tx: ident, $($tt: tt)*) => {
+        {
+            std::panic::set_hook(Box::new(|_| {
+                *crate::BACKTRACE.lock().expect("Mutex is poisoned") = Some(std::backtrace::Backtrace::capture());
+            }));
+
+            let ret = $($tt)*;
+
+            let _ = $tx.send(());
+
+            ret
```

Comment:
After `set_hook` you need to call `take_hook` to reset to the default hook handler

Reply:
I don't think we want to reset the default hook :thinking: Do you have a case ?

Reply:
If we panic outside the your system, we would lose the backtrace

Reply:
Done